### PR TITLE
Add tests storage management

### PR DESCRIPTION
**Pull request by lijinpei2008**

### CODE REVIEW

File: sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+            {
+                //talk to yugangw-msft, if the code doesn't work under dnx451 (same with net451)
+                X509Certificate2 cert = new X509Certificate2(certName, certPassword);
+                Handler = new System.Net.Http.WebRequestHandler();
+                ((WebRequestHandler)Handler).ClientCertificates.Add(cert);
+                ServicePointManager.ServerCertificateValidationCallback += (sender, certificate, chain, sslPolicyErrors) => { return true; };
+            }
+#endif
+            return Handler;
+        }
```
Comment (erich-wang) [inline sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs]: please remove it if not used

File: sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+
+        public static StorageAccountCreateParameters GetDefaultStorageAccountParameters(Sku Sku = null, Kind? Kind = null, string Location = null)
```
Comment (erich-wang) [inline sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs]: parameter name should be always camel style

File: sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+                Assert.AreEqual("value1", account.Tags["key1"]);
+                Assert.AreEqual("value2", account.Tags["key2"]);
+            }
+        }
+
+        public static AccountSasParameters ParseAccountSASToken(string accountSasToken)
+        {
+            string[] sasProperties = accountSasToken.Substring(1).Split(new char[] { '&' });
+
+            string ServiceParameters = string.Empty;
```
Comment (erich-wang) [inline sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs:143]: local variable names should be camel style

File: sdk/storage/Azure.Management.Storage/tests/Helpers/StorageTestsManagementClientBase.cs [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+namespace Azure.Management.Storage.Tests.Helpers
+{
+    [ClientTestFixture]
+    [NonParallelizable]
+    public abstract class StorageTestsManagementClientBase : RecordedTestBase<StorageAccountManagementTestEnvironment>
+    {
+        public bool IsTestTenant = false;
+
+        private const string ObjectIdKey = "ObjectId";
+        public string objectId { get; set; }
```
Comment (erich-wang) [inline sdk/storage/Azure.Management.Storage/tests/Helpers/StorageTestsManagementClientBase.cs]: remove it if not used

**Review by erich-wang — CHANGES_REQUESTED** [review]

File: sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs (hunk 1 above)
Comment (lijinpei2008) [reply sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs]: Have deleted.

File: sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs (hunk 2 above)
Comment (lijinpei2008) [reply sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs]: Update parameter style.

File: sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs (hunk 3 above)
Comment (lijinpei2008) [reply sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs:143]: Update parameter style.

File: sdk/storage/Azure.Management.Storage/tests/Helpers/StorageTestsManagementClientBase.cs (hunk 4 above)
Comment (lijinpei2008) [reply sdk/storage/Azure.Management.Storage/tests/Helpers/StorageTestsManagementClientBase.cs]: Have deleted.

**Pull request closed without merging**